app/src/controllers/administrate/projects/sagas.js

```diff
@@ -72,21 +71,25 @@ function* watchSetViewMode() {
   yield takeEvery(START_SET_VIEW_MODE, setViewMode);
 }
 
-function* addProject({ payload: projectName }) {
+function* addProject({ payload: { projectName, organizationSlug } }) {
   try {
-    const { projectKey } = yield call(fetch, URLS.addProject(), {
+    const projectKey = projectName.trim().replace(/\s+/g, '_');
+    yield call(fetch, URLS.addProject(), {
       method: 'post',
       data: {
         entryType: PROJECT_TYPE_INTERNAL,
-        projectName,
+        projectKey,
+        organizationId: organizationSlug,
```

Comment:
Back-end expect organizationId field.

Reply:
I see, but it a little bit confusing, may be we need to change naming on BE side?
Are they use this organisationId (organisationSlug) as id or slug to find proper organisation?
I think we can resolve this discussion, but please leave a comment with TODO to remember this case.

Reply:
I think, you are absolutely right. I do not understand, why we provide organizationId. Because we don't have this value.
And how it looks like it all. Is it the same as organizationSlug or not? Or we planning to have two of those. This is unclear for me.

Reply:
I guess we will have both.
It is a question to discuss with BE team, so also please leave a comment in Jira ticket. Hope we will consider this question on the backlog refinement :)

Reply:
Done!)